app/webpack.production.js

```diff
@@ -44,16 +45,13 @@ const rendererConfig = merge({}, common.renderer, config, {
       // appended to the index.html HEAD at compile time
       {
         test: /\.(scss|css)$/,
-        use: ExtractTextPlugin.extract({
-          fallback: 'style-loader',
```

Comment:
There's a [note in the README](https://github.com/webpack-contrib/mini-css-extract-plugin#advanced-configuration-example) for `mini-css-extract-plugin` that this should only be used for `production` builds and to stick to `style-loader` for development. I think we're good here to drop the `fallback` element given this is in the production config, but for reference this is how they do styling with one config:

```js
module.exports = {
  /* other things */
  module: {
    rules: [
      {
        test: /\.s?[ac]ss$/,
        use: [
          devMode ? 'style-loader' : MiniCssExtractPlugin.loader,
          'css-loader',
          'postcss-loader',
          'sass-loader',
        ],
      }
    ]
  }
}
```